--- Makefile.orig
+++ Makefile
@@ -6,9 +6,9 @@
 MANPREFIX = ${PREFIX}/share/man
 
 INCS = -I. -I${PREFIX}/include -I/usr/X11R6/include
-LIBS = -lc `pkg-config --libs xcb xcb-icccm xcb-keysyms xcb-ewmh`
+LDLIBS = -lc `pkg-config --libs xcb xcb-icccm xcb-keysyms xcb-ewmh`
 
-CPPFLAGS += -DVERSION=\"${VERSION}\" -DWMNAME=\"${WMNAME}\"
+override CFLAGS += -std=c99 -DVERSION=\"${VERSION}\" -DWMNAME=\"${WMNAME}\"
 
 DEBUG 	 = 0
 CFLAGS   += -std=c99 -pedantic -Wall -Wextra -Os ${INCS} ${CPPFLAGS} -DVERSION=\"${VERSION}\" 
@@ -45,8 +45,6 @@
 	@cp config.def.h $@
 
 ${WMNAME}: ${OBJ}
-	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
 
 clean:
 	@echo cleaning
